Answers.md

```diff
@@ -0,0 +1,14 @@
+1. If you saw this HTML: ```<div class="box box1 box2 box3"></div>``` which class has the most specificity weight?
+Box3
```

Comment:
I hope you understand why it is box3. Technically, all of the elements have the same specificity weight but if that is the case, then the last element will have the most weight. This is just a general rule of thumb.